docs/ctapipe_api/calib/index.rst

```diff
@@ -44,6 +48,19 @@ Submodules
 Reference/API
 =============
 
-.. automodapi:: ctapipe.calib
-    :no-inheritance-diagram:
-    :noindex:
+.. What follows is a *temporary* workaround to circumvent
+   various warnings of duplicate references caused by
+   calling automodapi on the camera package.
+
+ctapipe.calib Package
+---------------------
+
+Calibration
+
+.. raw:: html
+
+    <h3>Classes</h3>
```

Comment:
Why use HTML here? You can just use a third-level Restructured Text heading.  That will keep it working for output other than HTML

Reply:
I wanted to keep it consistent with the way `automodapi` outputs the headings. When using a third-level reST heading the color is the primary color and not the normal font color. I can change it to the third-level heading though, if you like.

Reply:
Okay, never mind, I fixed it with a `subsubsection` heading. Sorry, my mistake.